docs/_blog/announcing-anchor-outputs-channels.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Announcing Anchor Outputs Channels"
+description: "LDK v0.0.116 has added support for opening channels with anchor outputs."
+date: "2023-08-01"
+authors:
+  - Wilmer Paulino
+tags:
+  - Channels
+  - Anchor Outputs
+  - Fees
+---
+
+The `v0.0.116` release of LDK has added support for opening channels with anchor outputs! This has
+been a much requested feature by our users as it brings channels a few security and usability
+improvements.
+
+Lightning channels rely on pre-signed transactions that its participants can broadcast to the
+network if they wish to close the channel unilaterally, e.g., when their counterparty is offline.
+Before the anchor outputs feature, participants continually negotiated their commitment
+transaction’s fees based on the current blockspace demand to ensure a timely confirmation if they
+were to be broadcast. This fee negotiation unfortunately came with its own set of problems. If
+participants disagreed on the fees proposed for a channel, the channel became unusable due to a
```

Comment:
I don’t know if unusable is the most correct term, rather “the channel would have been automatically shutdown”. 

You might have issue when the channel become “stuck” / unusable because of wrong-estimation of the fee spikes reserve and the initiator do not have sufficient balance to add even an incoming HTLC output on the respective commitment transactions.

Quite nerdy as a distinction.